src/approaches/bilevel_planning_approach.py

```diff
@@ -105,3 +105,19 @@ def _get_current_predicates(self) -> Set[Predicate]:
         Defaults to initial predicates.
         """
         return self._initial_predicates
+
+    def get_option_model(self) -> _OptionModelBase:
+        """For ONLY an oracle approach or BEHAVIOR, we allow the user to get
```

Comment:
can you add assertions that we're in one of these two cases, both here and in the method below?